src/apps/FEMSWEEP-Sycl.cpp

```diff
@@ -0,0 +1,46 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-24, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "FEMSWEEP.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#if defined(RAJA_ENABLE_SYCL)
+
+#include "common/SyclDataUtils.hpp"
+
+#include <iostream>
+
+namespace rajaperf
+{
+namespace apps
+{
+
+void FEMSWEEP::runSyclVariant(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tune_idx))
+{
+  switch ( vid ) {
+
+    case Base_SYCL : {
+      break;
+    }
+
+    case RAJA_SYCL : {
+      break;
+    }
+
+    default : {
+      getCout() << "\n FEMSWEEP : Unknown Sycl variant id = " << vid << std::endl;
```

Comment:
Why not add the sycl version? It should be pretty straight forward.

Reply:
I would make an issue and push that off until a future pull request.